src/reducers/featureReducer.js

```diff
@@ -0,0 +1,66 @@
+import {
+  SET_NEXT_SLIDE,
+  SET_PREVIOUS_SLIDE,
+  SET_INIT_SLIDE,
+} from '../actions/types';
+
+const initialState = {
+  artList: [],
+  activeList: [],
+};
+
+/**
+ *  @param {function} state
+ * @param {function} action
+ *  @returns {Object} action obj
+ */
+function featureReducer(state = initialState, action) {
+  if (action.type === SET_INIT_SLIDE) {
```

Comment:
Why did you decide to use `if` instead of `switch`?

Reply:
No particular reason, I believe the team prefers to use switch as a convention I will adapt

Reply:
Please let's use switch. Makes the code cleaner.